src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -57,6 +57,7 @@ import { printElement } from '../../utils/utils';
 import { ColumnResize } from './utils/Resize';
 import Tooltip from '../tooltip/tooltip';
 import { Component, Vue } from '@/build/vue-wrapper';
+import DataTableEmpty from './DataTableEmpty';
```

Comment:
```suggestion
import DataTableEmptyActionable from './DataTableEmptyActionable';
```

Reply:
@ismael-teixeira remeber to import the agreeded naming convention to all the cases